.github/workflows/run_doublet-detection.yml

```diff
@@ -10,6 +10,8 @@ name: Run doublet-detection analysis module
 env:
   MODULE_PATH: analyses/doublet-detection
   AWS_DEFAULT_REGION: us-east-2
+  # use all samples for testing unless this is triggered by a PR, in which case only use a subset of samples for faster testing
+  SAMPLES: ${{ github.event_name != 'pull_request' && 'all' || 'SCPCS000435,SCPCS000133,SCPCS000134,SCPCS000135,SCPCS000136,SCPCS000001,SCPCS000024,SCPCS000050,SCPCS000101,SCPCS000250,SCPCS000168,SCPCS000216,SCPCS000124,SCPCS000298,SCPCS000481,SCPCS000484,SCPCS000514,SCPCS000490,SCPCS000502,SCPCS000246,SCPCS000616,SCPCS000632,SCPCS000758' }}
```

Comment:
I would move this down to the step where it is used. You can make an `env` block within that.

Reply:
For `SCPCS000133,SCPCS000134,SCPCS000135,SCPCS000136`  you should be able to use just one sample; the script will get all libraries that contain the sample, even if multiplexed.